react/src/components/CreateMapModal/CreateMapModal.tsx

```diff
@@ -95,99 +100,55 @@ const CreateMapModal = ({ isOpen, toggle }: CreateMapModalProps) => {
-                  as={Input}
-                  invalid={touched.name && !!errors.name}
-                />
-                <ErrorMessage
-                  name="name"
-                  component="div"
-                  className="invalid-feedback"
                 />
-              </FormGroup>
-              <FormGroup>
-                <Label for="description">Description</Label>
-                <Field
+                <FormikTextarea
                   name="description"
-                  id="description"
-                  className="col-sm-11"
-                  as={Input}
-                  invalid={touched.description && !!errors.description}
+                  label="Description"
+                  required
                 />
-                <ErrorMessage
-                  name="description"
-                  component="div"
-                  className="invalid-feedback"
-                />
-              </FormGroup>
-              <FormGroup>
-                <Label for="system_file">Custom File Name</Label>
-                <div className="input-group">
-                  <Field
-                    name="system_file"
-                    id="system_file"
-                    as={Input}
-                    className="col-sm-8"
-                    invalid={touched.system_file && !!errors.system_file}
-                  />
-                  <span className="input-group-text col-sm-3">.hazmapper</span>
+                <div className={`${styles['flexContainer']}`}>
+                  <span>
+                    <FormikInput
+                      name="system_file"
+                      label="Custom File Name"
+                      required
+                      className={`${styles['input-custom-size']}`}
+                    />
+                  </span>
+                  <span className={`${styles['hazmapper-custom']}`}>
+                    .hazmapper
+                  </span>
                 </div>
-                {/* Alt solution to render error message bc input group was causing text to not display properly */}
-                {touched.system_file && errors.system_file && (
-                  <div className={`${styles['custom-error-message']}`}>
-                    {errors.system_file}
-                  </div>
-                )}
-              </FormGroup>
-              {/* TODO_REACT: This part will change once the FileBrowser component is added. https://tacc-main.atlassian.net/browse/WG-208*/}
-              <FormGroup className="row align-items-center">
-                <Label className="col-sm-4">Save Location:</Label>
-                <div className="col-sm-8 text-primary">
-                  {userData ? `/${userData.username}` : 'Loading...'}
+                <div className={`${styles['flexContainer-alt']}`}>
+                  <label htmlFor="save-location-label">Save Location:</label>
+                  <span className="text-primary">/{userData?.username}</span>
```

Comment:
**Outdated class name.**

The `text-primary` is a holdover from Core-Portal. Please do not use it.